ui/DesignSystem/Component/SourceBrowser/CommitTeaser.svelte

```diff
@@ -1,7 +1,11 @@
 <script>
```

Comment:
Is this component used anywhere?

Reply:
Yeah, it's what is shown above source files for eg.